\documentclass[9pt,twocolumn,twoside]{gsajnl}
% Use the documentclass option 'lineno' to view line numbers
\usepackage{booktabs}
\usepackage{siunitx}
\usepackage{adjustbox}
\usepackage{geometry}
\articletype{gos} % article type

\title{Hill-Robertson Interference Causes Reduced Genetic Diversity on a Young Plant Y-chromosome}

\author[$\ast$,$\dagger$,1]{Josh Hough}
\author[$\dagger$]{Wei Wang}
\author[$\dagger$]{Spencer C.H. Barrett}
\author[$\dagger$]{Stephen I. Wright}

\affil[$\ast$]{Department of Plant Sciences, University of California, Davis}
\affil[$\dagger$]{Department of Ecology and Evolutionary Biology, University of Toronto}

%\affil[$\dagger$]{Author two affiliation}
%\affil[$\ddagger$]{Author three affiliation}
%\affil[$\S$]{Author four affiliation}
%\affil[$\ast\ast$]{Author five affiliation}

\keywords{Sex Chromosome Evolution; Nucleotide Diversity; Recombination; Deleterious Mutations}

\runningtitle{Plant sex chromosome evolution} % For use in the footer

\correspondingauthor{Josh Hough}

\begin{abstract}
X and Y chromosomes differ in effective population size ($N_{e}$), rates of recombination, and exposure to natural selection, all of which can affect patterns of genetic diversity. On Y chromosomes with suppressed recombination, natural selection is expected to eliminate neutral variation and lower the $N_{e}$ of Y compared to X chromosomes or autosomes. However, non-selective factors including female-biased sex ratios and high variance in male reproductive success can also reduce Y-linked $N_{e}$, making it difficult to infer the causes of low Y-diversity from natural populations. Here, we investigate the factors affecting levels of polymorphism during sex chromosome evolution in \textit{Rumex hastatulus} (Polygonaceae), a dioecious plant with young sex chromosomes. Strikingly, we find that neutral diversity for genes on the Y is on average 2.1\% of the value for their homologues on the X, corresponding to a chromosome-wide reduction of 99.2\% compared to the neutral expectation. We demonstrate that the magnitude of this diversity loss is inconsistent with a reduced male $N_{e}$ caused by neutral processes including female-biased sex ratios and high variance in male reproductive success. Instead, using forward simulations and estimates of the distribution of fitness effects of deleterious mutations, we show that diversity loss on the Y can be explained by purifying selection acting over a large number ($\geq$ 800 Kb) of genetically-linked sites. Our results are in agreement with theory on "interference selection", and provide evidence that this effect can substantially reduce nucleotide diversity on a young Y chromosome. Given the relatively recent origin of \textit{R. hastatulus} sex chromosomes, our results imply that Y-chromosome degeneration in the early stages may be largely driven by such interference effects rather than by positive selection for gene silencing followed by neutral genetic drift.
\end{abstract}

\setboolean{displaycopyright}{true}

\begin{document}

\maketitle
\thispagestyle{firststyle}
\marginmark
\firstpagefootnote
\correspondingauthoraffiliation{jhough@ucdavis.edu}
\vspace{-11pt}

\section*{Introduction}

\lettrine[lines=2]{\color{color2}M}{}orphologically distinct sex chromosomes have evolved multiple times independently in both plants and animals \citep{westergaard1958,ohno1967,bull1983,charlesworth1991,charlesworth2015plant}. Despite clear biological differences between these kingdoms, X and Y chromosomes in both lineages have undergone similar genetic changes. In both groups, for example, the loss of recombination between X and Y chromosomes is associated with an accumulation of deleterious mutations and a gradual loss of genes from the Y chromosome \citep{hough2014,bergero2015,bachtrog2013NRG}. In some species, such genetic deterioration of the Y chromosome has also led to the evolution dosage compensation of the X chromosome \citep{charlesworth1996CB,muyle2012,mank2013sex,papadopulos2015}. The independent evolution of these phenomena in such taxonomically distant groups as mammals and flowering plants suggests that general evolutionary mechanisms may be involved, but inferring the causes of molecular evolution and patterns of polymorphism in genomic regions that lack recombination is a longstanding challenge for both theoretical and experimental biologists \citep{charlesworth1978,feldman1980evolution,barton1995general,charlesworth1996CB,otto1997deleterious,charlesworth2000degeneration,mcvean2000effects}.

One fundamental difference between X and Y chromosomes is that there are 1/3 as many Y-linked gene copies as X-linked ones in a diploid population. Therefore, genes on the Y chromosome are expected to experience an effective population size ($N_{e}$) that is 1/4 that of autosomal genes, whereas the  $N_{e}$ for genes on the X chromosome should be 3/4 that of autosomal genes (assuming an equal number of reproducing females and males). The lowered $N_{e}$ of the Y chromosome implies that the equilibrium level of neutral polymorphism - proportional to the product of $N_{e}$ and the neutral mutation rate, $\mu$ - should be lower for Y-linked genes than for their X-linked counterparts. In the absence of recombination, genes on the Y chromosome are also expected to be in strong linkage disequilibrium, making them vulnerable to diversity loss due to selection against strongly deleterious mutations (background selection; \citep{charlesworth1993effect}) and selective sweeps of strongly beneficial mutations (genetic hitchhiking; \citep{smith1974hitch}). Furthermore, the build-up of genetic associations among selected mutations (high fitness covariance) on the Y means that selection will act non-independently across the chromosome, such that selection at any given focal site "interferes" with selection at the sites with which it is linked \citep{hill1966HReffect}. A large body of work has now shown that in theory such "selective interference" effects can substantially reduce both the efficacy of selection and the equilibrium level of neutral variability \citep{fisher1930genetical, muller1964relation, hill1966HReffect, mcvean2000,KaiserCharlesworth,good2014genetic}, with the magnitude of the effect depending strongly on the number and density of sites experiencing selection . These arguments all suggest that large non-recombining Y chromosomes should harbor a lower amount of neutral variability than predicted based on the number of Y chromosomes in a population (1/4 that of autosomes), and highlight that estimating the number of selected sites is a key test for determining the likely mechanism involved in driving diversity loss on a nonrecombining Y chromosome. However, the relative importance of neutral and selective factors for reducing chromosome-wide levels of diversity is not well-understood.

In addition to reduced diversity arising from selection, in species with female-biased sex ratios or extensive male-male competition, high variance in male reproductive success is also expected reduce the $N_{e}$ experienced by genes on the Y chromosome \citep{caballero1995,charlesworth2001,laporte2002,pool2007,ellegren2009}, suggesting that inferences about the effects on diversity of positive of purifying selection need to be distinguished from these neutral processes. Because variance in male reproductive success differentially affects the $N_{e}$ of X, Y, and autosomal chromosomes \citep{kimura1964number,nomura2002effective}, evidence for this can therefore be obtained by comparing levels of silent site variability on X and Y chromosome, relative to values on autosomes. For example, high variance in male reproductive success reduces male $N_{e}$, which reduces the Y/A diversity ratio compared to the neutral expectation of 1/4, but it also increases the X/A ratio. Based on such comparisons, studies in humans, for example, have suggested that the inflated X/A ratio is due to a historical excess of breeding females compared to males \citep{hammer2008sex} (but see \citep{bustamante2009,hammer2010,cotter2016genetic}).

Despite widespread interest in determining the evolutionary factors affecting neutral diversity on sex chromosomes \citep{ellegren2011,bachtrog2013NRG}, we know very little about the influence of sex ratio variation or linked selection in determining levels of diversity on more recently evolved sex chromosomes. The time scales over which these different effects are likely to be important are therefore not well understood. In humans, estimates of Y-linked diversity are considerably lower than predicted under neutral models, and simulations suggest that levels of diversity are consistent with strong purifying selection \citep{Wilsonsayres2014}. However, given that human sex chromosomes evolved from autosomes \textasciitilde 200 million years ago (MYA) \citep{lahn1999four,ross2005dna}, it is not clear whether purifying selection might have similarly strong effects on Y chromosomes that evolved \textit{do novo} from autosomes over much more recent evolutionary time (e.g., within the last \textasciitilde 20 MYA in the case of plants \citep{charlesworth2015plant}).

On the one hand, simulations of strong selection models (background selection and genetic hitchhiking) suggest that these processes may have the greatest effects during the earliest stages of sex chromosome evolution, before the Y has lost many of its genes \citep{bachtrog2008temporal}. Moreover, theory suggests that even weak purifying selection, if acting across a large number of genetically-linked sites, can generate substantial deviations from neutrality, whereas classic background selection theory breaks down in such cases \citep{mcvean2000, comeron2002population, KaiserCharlesworth, good2014genetic}. Given that a large number of selected sites are likely to be in linkage disequilibrium on a recently evolved Y chromosome, such "interference selection" \textit{sensu} \citep{good2014genetic} is \textit{a priori} likely to have strong effects on young plant sex chromosomes. On the other hand, if there has been widespread gene silencing during the early stages of Y degeneration, as found on the \textit{Drosophila miranda} neo-Y chromosome \citep{zhou2012chromosome}, then Y-linked diversity loss might be low in younger sex chromosome systems. In particular, if gene silencing occurs early during Y chromosome degeneration, few sites are expected to be experiencing purifying selection, with degeneration primarily driven by neutral genetic drift rather than inefficient selection. Distinguishing the relative roles of selective interference and neutral drift during Y chromosome evolution is a major open issue.

To investigate the factors affecting nucleotide diversity in the early stages of sex chromosome evolution, we analyzed neutral polymorphism levels on X, Y, and autosomal chromosomes in the plant \textit{R. hastatulus }(Polygonaceae). This species is a dioecious annual with heteromorphic X and Y chromosomes that originated \textasciitilde 15 MYA \citep{quesada2011,grabowska2015,navajas2005}, making Y chromosomes in this species over 100 million years younger than the highly degenerated Y chromosomes in mammals \citep{lahn1999,ross2005dna}. \textit{Rumex hastatulus} has also received particular attention because of the occurrence of an interesting polymorphic in sex chromosome system, in which both XY and $XY_{1}XY_{2}$ males occur in geographically distinct populations ( so-called "chromosomal races") \citep{smith1963mechanism}. The $XY_{1}XY_{2}$ sex chromosome system in this species is thought to have originated through an X-autosome fusion, with the XY system maintaining the ancestral chromosome complement \citep{smith1964evolving}. Despite the recent origin of sex chromosomes in both races, there is evidence that both Y's have undergone gene loss and functional deterioration \citep{hough2014}. Here, to simplify our comparison of polymorphism levels on X, Y, and autosomes, and to ensure that our diversity estimates are not biased by non-equilibrium conditions or other confounding factors arising from the X-autosome fusion in the $XY_{1}XY_{2}$ system, we focus only on the XY system.

Of particular importance for our study, \textit{R. hastatulus} populations have been found to consistently exhibit female-biased reproductive sex ratios, with a mean sex ratio of $N_{f}/(N_{m}+N_{f})=0.6$ \citep{pickup2013influence}. Female-biased sex ratios are not uncommon in dioecious plants with heteromorphic X and Y chromosomes \citep{field2013comparative,hough2013evolutionarily}, and their occurrence in \textit{R. hastatulus} provides an opportunity to test both the demographic and selective contributions to changes in sex-linked variability during sex chromosome evolution.

\section*{Materials and Methods}
\subsection*{Population Samples and Sex-Linked Genes}
We analyzed sex-linked and autosomal genes identified from Illumina RNA sequence data from 12 population samples (1 male and 1 female from each of 6 populations). Samples were collected in 2010 from throughout the native range of \textit{R. hastatulus} (locations in Table S1), and plants were grown in the glasshouse at the University of Toronto from seeds collected from open-pollinated females. We extracted RNA from leaf tissue using Spectrum Plant Total RNA kits (Sigma-Aldrich). Isolation of mRNA and cDNA synthesis was conducted according to standard Illumina RNAseq procedures, and sequencing was conducted on two Illumina HiSeq lanes with 150-bp end reads at the Genome Quebec Innovation Center. Reads from these samples were mapped to the \textit{R. hastatulus} reference transcriptome \citep{hough2014} using the Burrows–Wheeler Aligner \citep{li2010fast}, followed by Stampy \citep{lunter2011stampy}. We used Picard tools (http://picard.sourceforge.net) to process mapping alignments for the Genome Analysis Toolkit \citep{mckenna2010genome} variant calling software, and subsequently removed genes with low coverage (<10x) and low Phred Quality Scores <20. The population samples analyzed here were previously reported in Hough et al. (2014), where they were used to validate the ascertainment of sex-linked genes identified through segregation analysis, and raw sequences are available from the GenBank Short Read Archive under accession no. SRP041588. Here, to consider sex linked genes that were identified in all of our sequenced population samples, we focused on the previously described set of 460 X/Y genes for which a Y homolog was found in both the Texas and North Carolina races (i.e., X/Y genes where the Y copy was inferred to be on the $Y_{1}$ chromosome).

\subsection*{Autosomal Genes}
In evaluating evidence for nucleotide diversity differences between X and Y chromosomes, it is important to distinguish between reduced Y-linked diversity, and the possibility that X-linked diversity is elevated above the level predicted from a neutral model. To do this, we normalized our sex-linked diversity estimates by autosomal diversity, and compared empirical X/A and Y/A nucleotide diversity ratios to those predicted from neutral models and from simulations (described below). Because the criteria for ascertaining autosomal loci in Hough et al. (2014) were based on identifying four segregating SNPs per locus, and since this set of genes is likely to be higher in diversity than the average autosomal gene, here we instead used the larger set of all non-sex linked (putatively autosomal) genes as our autosomal reference. We filtered genes in this set to remove any genes that may have been sex-linked but were not identified as such by Hough et al.'s conservative ascertainment criteria. In particular, we removed: (i) any genes in which there was evidence for at least one SNP with a sex-linked segregation pattern, (ii) any genes where SNPs showed fixed heterozygosity in males and fixed homozygosity in females, (iii) genes with less than 10X coverage or greater than 100X coverage from independently obtained genomic coverage (Beaudry et al., in prep) data (to filter out duplicates or genes with highly repetitive sequences), and (iv) any genes containing SNPs with large (>0.4) allele frequency differences between males and females. Finally, we removed genes with fewer than 100 synonymous sites to avoid biasing our results toward genes that may have been particularly short due to assembly problems. This filtering resulted in a final set of 12,356 autosomal genes.

\subsection*{Phasing X and Y alleles}
To estimate polymorphism for X and Y sequences separately in males, it is necessary to infer the phase of SNPs in sex-linked transcripts in males. In previous work, phasing alleles on \textit{R. hastatulus} sex chromosomes was achieved using segregation analysis from a genetic cross. Here, to phase SNPs from population samples where such segregation data was unavailable, we used HAPCUT \citep{bansal2008hapcut}, a maximum-cut based algorithm that reconstructs haplotypes using sequenced fragments (Illumina read data) from the two homologous chromosomes to output a list of phased haplotype blocks containing the SNP variants on each chromosome. Because the resulting haplotype blocks produced by HAPCUT contained SNPs that were phased relative to each other, but not designated to either the X or Y chromosome, we assigned individual variants to X or Y by independently identifying fixed X-Y differences within each haplotype block (i.e., sites where all females were homozygous, and all males were heterozygous). Identifying such fixed differences within phased haplotype blocks enabled us to then infer the correct phase (X or Y) of the polymorphisms from HAPCUT’s output. In particular, this was done by matching the phase of fixed X-Y differences with their neighboring polymorphic sites: when a fixed X-Y difference occurred in the same phased haplotype block as a polymorphic site, then the polymorphic variants in that block were assigned to either X or Y based on the known phase of the fixed difference with which they were matched. SNPs that were identified outside of phased blocks, or in blocks without fixed X-Y differences, were recorded as missing data. Finally, we filtered out SNPs with quality scores < 60, and those within a distance of 10bp or less from indels. This filtering procedure resulted in alignments of X and Y sequences for 372 sex-linked genes.

We further validated the results of HAPCUT’s allele phasing by comparing the accuracy of this method with the phasing-by-segregation method that was conducted in Hough et al. (2014). To do this, we first phased the sequence data from parents and their progeny using HAPCUT’s algorithm (using the same parameters as for the population data), and then identified cases where SNPs were inferred on the Y chromosome by HAPCUT, but where the true level polymorphism, obtained from the genetic cross, was zero. We identified 7 \% of sex-linked genes that either had phasing errors genotyping errors. This corresponds to a SNP error rate estimate of 1.7 x 10-4. Note that this rate is very low relative to population-based estimates of polymorphism on the X and autosomes (Table 1), and therefore should have minimal effects on our estimation of the X/A ratio. However, because this rate is high relative to the expected level of polymorphism on the Y chromosome, we conducted a further filtering step for Y-linked SNPs and identified false positive SNP calls arising from: (i) phasing errors caused by gene duplicates (more than two haplotypes), (ii) polymorphisms around indels, and (iii) genotyping errors caused by low Y-expression. This final filtering was conducted by manually checking each individual putative polymorphism on the Y chromosome using IGV \citep{robinson2011integrative}.

\subsection*{Estimating nucleotide diversity on sex chromosomes and autosomes}
For each locus in our analysis, we calculated Watterson’s (1975) estimator of the population parameter $\theta=4N_{e}\mu$, where $N_{e}$ is the effective population size, and $\mu$ is the mutation rate \citep{watterson1975}, using a modified version of the Perl program Polymorphurama \citep{bachtrog2006}. To compare sex-linked and autosomal loci, we calculated the average value of $\theta$, weighted by the number of synonymous sites in each gene (\X Figure 2). We obtained 95\% confidence intervals for X/A and Y/A ratios by bootstrapping per gene using the BCa method \citep{efron1994} implemented in the Boot package in R \citep{canty2012boot}, and calculating X/A and Y/A on each iteration for 20000 replicates each. Bootstrapping was conducted on the final filtered set of 173 sex-linked, and 12355 autosomal genes.

Note that the lack of recombination on the Y chromosome implies that statistical assumptions about independence across loci are violated, suggesting that the true uncertainty in the Y/A estimate may be wider than implied by bootstrapping. To address this, we also used a maximum likelihood approach, implemented in a modified version of the MLHKA software \citep{wright2004hka}, to independently estimate a credibility interval for the Y/A ratio (Figure S1). Because of the thousands of genes involved, a likelihood method incorporating divergence to control for heterogeneity in mutation rate was not feasible, as this would require maximizing the likelihood estimate of the mutation rate for each locus independently. However, previous analysis of divergence data does not suggest important chromosomal differences in synonymous divergence contributing to diversity heterogeneity (see Results). Therefore, we assumed no heterogeneity in mutation rate, no recombination between Y-linked genes, and free recombination between autosomal loci. Our model thus had two parameters: $\theta_{autosomal}$ and \textit{f}, the ratio of effective population size of the Y chromosome to autosomes. We varied both parameters and evaluated the likelihood for \textit{f} from 0.001 to 1, and $\theta_{autosomal}$ per base pair from 0.001 to 0.01. In particular, we evaluated the likelihood of $\theta_{autosomal}$ and \textit{f}, given the observed number of synonymous segregating sites for each \textit{i} of \textit{n} autosomal loci ($S_{i}$), the number of segregating sites on the Y chromosome ($S_{y}$), and the total number of sites per locus ($L_{i}$ and $L_{y}$). The likelihood of ($\theta_{autosomal}$, f) is therefore given by:

\begin{equation}
L \propto  P(S_{Y}, L_{Y} | f, \theta_{a})\prod_{i=1}^{n}P(S_{i}, L_{i} | \theta_{a})  \label{eq:L}
\end{equation}

\subsection*{Neutral predictions and the effect of sex ratio bias on diversity}
To test whether our estimated levels of diversity on X, Y and autosomal chromosomes could be explained by neutral processes, including female-biased population sex ratios or high variance in male reproductive success, we compared our normalized X/A and Y/A diversity estimates to neutral predictions for a range of values for the ratio of effective population sizes of females and males. Because we were primarily interested in determining the parameter space within which a reduced male effective population size could result in Y/A and X/A diversity ratios consistent with our data, we tested the fit of our data to predictions across the full range of $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m})$ (0.1 to 0.9; we excluded all-female or all-male cases). For autosomal and sex-linked effective population sizes given by:

\begin{equation}
N_{e_{A}} = \frac{4N_{e}_{m}N_{e}_{f}}{N_{e}_{m}+N_{e}_{f}} \label{eq:NeA}
\end{equation}

\begin{equation}
N_{e_{X}} = \frac{9N_{e}_{m}N_{e}_{f}}{4N_{e}_{m}+2N_{e}_{f}} \label{eq:NeX}
\end{equation}

\begin{equation}
N_{e_{Y}} = \frac{N_{e}_{m}}{2} \label{eq:NeY}
\end{equation}

\citep{wright1931evolution}, the corresponding expected X/A and Y/A ratios an be expressed as:

\begin{equation}
\frac{N_{e_{X}}}{N_{e_{A}}} = \frac{9 (-1 + r)}{8 (1 - r) (-2 + r))} \label{eq:XA}
\end{equation}

and
\begin{equation}
\frac{N_{e_{Y}}}{N_{e_{A}}} = \frac{1}{8r} \label{eq:YA}
\end{equation}

respectively, where $r = N_{e}_{f}/(N_{e}_{f} + N_{e}_{m})$.  Note that when $N_{e}_{f} = N_{e}_{m}$ ($r=0.5$) then, $N_{e_{X}}/N_{e_{A}} = 0.75$, and $N_{e_{Y}}/N_{e_{A}} = 0.25$ as in the standard neutral model \citep{wright1931evolution}. With a female-biased ratio, however, the expected X/A ratio can become greater than 1, approaching 1.125 in the limit as r\rightarrow 1 \citep{caballero1995} (Figure 1).

\begin{figure}[htbp]
\centering
\noindent
\includegraphics[width=\linewidth]{figure1.jpg}
\caption{\textbf{A}. Expected effective population sizes, relative to $N$,  for genes on autosomes (A), X chromosomes (X), and Y chromosomes (Y) as a function of $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m})$. \textbf{B}. The corresponding X/A and Y/A ratios of diversity predicted at equilibrium (where $\theta=4N_{e}\mu$), assuming equal neutral mutation rates for sex-linked and autosomal genes.
}
\end{figure}

\subsection*{Simulations of purifying selection}
To study the effects of purifying selection on expected levels of Y-chromosome diversity, we conducted forward-time simulations of haploid Y chromosomes using the software SFSCODE \citep{hernandez2008flexible}. We first estimated the distribution of fitness effects of deleterious mutations from our polymorphism data for X-linked genes using the method of \citep{keightley2007joint}, which fits a gamma distribution of negative selection coefficients to the observed frequency distribution of nonsynonymous and synonymous polymorphisms. We then used this estimated gamma distribution to parameterize the simulations, initializing them with our estimated $\theta$ from autosomal genes, adjusted to reflect the expected neutral reduction in $N_{e}_{Y}$ for a sex ratio of $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m}) = 0.6$. To match our sample size and the number of synonymous sites sampled from our data (see Supporting Information), the simulations sampled 6 haploid chromosomes, and the genome sequence contained 45,331bp of linked neutral sequence from which we calculated silent site diversity. To examine the expected reduction in diversity as a function of the number of selected sites ($L_{selected}$), we ran simulations over a range of values of $L_{selected}$, up to a maximum of $5x$10^{6}$ (Figure 3A). To estimate $L_{selected}$, we calculated the approximate likelihood of our observed data based on the proportion of simulations in which synonymous diversity, $\pi_{simulated}$, was less than 0.0001 from our empirical estimate, $\pi_{observed}$ (Figure 3B).

\section*{Results and Discussion}

\subsection*{Extensive loss of Y-chromosome diversity}
Our analysis of polymorphism levels across the genome in the dioecious plant \textit{R. hastatulus} reveals that neutral diversity on the Y chromosome is substantially lower than expected from a standard neutral model. In particular, neutral diversity for Y-linked genes is on average \textasciitilde 2.1\% of the value for their homologues on the X chromosome. Taking 1/4 of the mean autosomal diversity as the equilibrium expectation for $\theta_{Y}$ under neutrality \citep{wright1931evolution}, this corresponds to a chromosome-wide reduction of 99.2\% . Note that by normalizing X and Y diversity estimates by autosomal diversity levels, our results indicate that the difference between X and Y homologues is not due to an elevation of diversity on the X chromosome, but a Y-specific reduction. Interestingly, our results also suggest an elevation of X-linked diversity ($X/A=0.85$) relative to the neutral prediction ($X/A=0.75$), though the confidence intervals on this estimate are wide, ranging from 0.74 to 0.95 (Figure 2). Given the empirically-estimated sex ratio of 0.6 in \textit{R. hastatulus}\citep{pickup2013influence}, however, the elevation we observed is not unexpected; the neutrally-predicted X/A ratio for a sex ratio of 0.6 is $\approx 0.8$ (Figure 2; Equation 4). Although our estimates of diversity have not been normalized by divergence, previous work has shown that the average synonymous substitution rate, between \textit{R. hastatulus} and the non-dioecious outgroup \textit{R. bucephalophorus} is not significantly different for sex-linked (0.2016) and autosomal genes (0.219), and we found no evidence for significant differences in substitution rate between Y and X chromosomes \citep{hough2014}. It is therefore unlikely that our results are caused by mutation rate differences between sex-linked and autosomal genes.

\begin{table}[htb]
\centering
\caption{Observed and neutrally-expected silent site diversity on sex chromosomes and autosomes in \textit{R. hastatulus}}
\begin{tabular}{ccccccccc}
\textbf{} & \multicolumn{2}{l}{\textbf{Observed}} & \multicolumn{3}{l}{\textbf{Expected}} \\

chr & $\theta$ & $/\theta_{A}$ & $\theta$ & $/\theta_{A}$  \\
\midrule
A & 0.006 & 1 & N/A & 1 \\
X & 0.0047 & 0.85 & 0.0045 & 0.75 \\
Y & $\num{e-4}$ & 0.002 & 0.0015 & 0.25 \\
\addlinespace

\bottomrule
\end{tabular}
\end{table}

\subsection*{Female biased sex ratios and variance in male offspring number}
The occurrence of female-biased sex ratios in \textit{R. hastatulus} is expected to lower Y diversity through a reduction in male $N_{e}$ and thus a reduction the $N_{e}$ of the Y chromosome. Male $N_{e}$ could be further reduced by high variance in male reproductive success, which is not unusual in annual wind-pollinated plants such as \textit{R. hastatulus} that commonly exhibit extensive phenotypic plasticity in plant size and flower production. Given that male plants in this species produce large amounts of pollen, and female flowers are uniovulate, there may indeed be strong competition among males to fertilize females. Such competition should increase levels of X-linked diversity to a level that is close to (or even higher than) levels of autosomal diversity \citep{caballero1995}, while simultaneously reducing Y-linked diversity. In common with most flowering plants, we do not have marker-based estimates of variance in male reproductive success in \textit{R. hastatulus}. We therefore tested whether an overall reduction in male $N_{e}$, arising either from high variance in male reproductive offspring number and/or a female biased population sex ratio, could explain our observed Y/A and X/A ratios, by comparing our data to neutral predictions across the full range of values for $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m})$.

As shown in Figure 2, the expected reduction in the Y/A ratio across the full range of $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m})$ is substantially higher than our observed reduction. Indeed, the lower limit for the Y/A ratio in a neutral model is 1/8, such that even in the extreme case where $N_{e}_{f}/(N_{e}_{f} + N_{e}_{m}) = 0.9$, the expectation for the Y/A ratio is $\approx 0.14$, which is substantially higher than our observed estimate ($Y/A= 0.002$) (Table 1; Figure 2; Figure S3). Moreover, such large reductions in $N_{e}_{m}$ also predict levels of X/A diversity that are not significantly larger than what we observed (Figure 2). Our results therefore indicate that, although reduced male $N_{e}$ arising from sex-biased demography is expected to contribute to reduced Y chromosome polymorphism, it is insufficient to explain the Y/A reduction we observed.

\begin{figure}[htbp]
\centering
\noindent
\includegraphics[width=\linewidth]{figure2.jpg}
\caption{Observed and predicted X/A and Y/A ratios of neutral diversity. Predictions are shown for increasing values of $N_{e}_{f}/N_{e}_{m}$. Observed estimates of X/A and Y/A were calculated as the average $\theta$ across genes, weighted by the number of synonymous sites per gene. Confidence intervals were calculated by bootstrapping (20000 replicates) using the BCa method (Efron 1987) implemented in the Boot package in R (Canty and Ripley 2012; R Core Team 2011), and using maximum likelihood for the Y-chromosome (see Methods).
}
\label{fig:spectrum}
\end{figure}


\subsection*{Purifying selection}
Selection against strongly deleterious mutations is expected be an important factor reducing genetic variability in regions with low recombination \citep{charlesworth1993effect, charlesworth1996background}. The effects of such “background selection” (BGS) have been well studied theoretically \citep{charlesworth1997effects, nordborg1996effect, kim2000joint} and the theory has been used to explain patterns of genetic diversity across genomes \citep{comeron2014background} and plays a central role in explanations for the extensive diversity loss on Y chromosomes in mammals \citep{Wilsonsayres2014} and Drosophila \citep{mcallister1999, charlesworth1996CB}. Under background selection theory, the predicted loss of diversity is modeled as modeled as a reduction in the equilibrium $N_{e}$ such that $\pi \approx 4f_{0}N_{e}\mu$, where $f_{0}=e^{-\frac{U}{sh+r}}$, $U$ is the mutation rate to strongly deleterious variants, $s$ and $h$ are selection and dominance coefficients, and $r$ is the recombination rate. This yields the expected value of $\pi$ relative to that in the absence of BGS ($\pi_{0}$): $E[\pi]= \pi_{0}e^{\frac{-U}{2sh+r}}$, where , $\pi_{0}=\theta=4N_{e}\mu$. Because only reproducing males bearing Y chromosomes are relevant for determining the level of Y-linked diversity (i.e., $N_{e}_{Y} = N_{e}{m}/2$), the expected reduction in diversity is obtained in this haploid model by replacing $N_{e}$ above with number of breeding males, $N_{e}_{m}$. Assuming that $N_{e}_{m}$ is 0.4 of the total $N_{e}$ estimated from autosomes because of the female biased sex ratio \citep{pickup2013influence} and using $s$=0.003 as estimated from the distribution of fitness effects of deleterious mutations on the X chromosome, the corresponding expected value is $E[\pi]= 6x10^{-5}$ for a mutation rate of $10^{-8}$ per nucleotide per generation. Although there is considerable uncertainty in these parameters, the calculation yields a value of diversity that is 67\% lower that what we observed (Table 1).

Importantly, BGS assumes independence among sites, and the theory breaks down when many linked sites are subject to selection at the same time, because the behavior of selected alleles interferes with the action of selection at linked sites \citep{good2014genetic,KaiserCharlesworth}. As there are no analytical formulae for predicting the outcome of selection on many linked alleles experiencing selection and drift, however, simulations are fundamental to understanding this selective interference in realistic situations. We therefore conducted forward simulations of purifying selection using our estimated parameters for the distribution of fitness effects of deleterious mutations, and tested whether interference selection could result in a level of Y diversity at neutral sites similar to the level we observed. We found that models of purifying selection acting over a large number of linked selected sites resulted in an expected loss of diversity at neutral sites that was a remarkably good fit to our data (Figure 3). In addition, and consistent with previous studies, our simulations also suggest a saturating loss of diversity with an increasing number of selected sites \citep{KaiserCharlesworth}.

Although we initialized our simulations with selection parameters estimated from the distribution of fitness effects of deleterious mutations, recent work has shown that, unlike with BGS, evolutionary dynamics in the “interference selection regime” are insensitive to differences in strength of selection, with expected diversity loss instead determined primarily by the population fitness variance over the non-recombining region, which is in turn expected to increase with the number of linked loci under selection \citep{good2014genetic}. Estimating the number of selected loci is therefore key to determining whether diversity on the Y chromosome may be affected by interference selection. Our maximum likelihood estimates of the number of selected sites, $L$, from our simulations, suggest that the number of sites subject to selection required to explain the diversity loss from the \textit{R. hastatulus} Y chromosome is ($\approx 800 kb$) (Figure 3).

\begin{figure}[t!]
\centering
\noindent
\includegraphics[width=\linewidth]{figure3.jpg}
\caption{Simulation results of the level of neutral diversity $pi_{syn}$ vs. the number of selected sites (top panel) with the dashed line corresponding to our point estimate for the level of neutral diversity on the Y chromosome, and the relative likelihood curve for the selected sites (bottom panel), with the dashed line corresponding to our approximate 95\% credibility interval.
}
\label{fig:spectrum}
\end{figure}

Although our analyses of purifying selection models are consistent with our observed reduction in Y-linked diversity, it is also possible that selective sweeps have contributed. We have not formally excluded this possibility. However, if patterns of molecular evolution on the Y are driven primarily by interference selection, as our results suggest, then a further implication is that beneficial mutations should suffer a reduction in their probability of fixation, limiting the scope for sweeps to drive down diversity. Nonetheless, there is abundant evidence that “soft sweeps” are important determinants of patterns of nucleotide diversity \citep{messer2013population}, and it is possible that they have contributed to the extremely low levels of Y-linked variability that we observed.

\section*{Conclusions}
The non-recombining region of a Y chromosome produces evolutionary dynamics that are similar to a haploid asexual population, with the lack of recombination resulting in build up of genetic associations among selected sites, a loss in the efficiency of selection, and an increase in population fitness variance \citep{fisher1930genetical, muller1964relation, hill1966HReffect, mcvean2000,KaiserCharlesworth,good2014genetic}. Our study of neutral diversity levels on the relatively young \textit{R. hastatulus} sex chromosomes provides clear evidence for an extensive loss of neutral diversity on the Y chromosome. Whereas neutral models of sex-biased demography were unable to explain the magnitude of diversity loss on this chromosome, forward population genetic simulations suggest that purifying selection acting over a large number of selected sites results in an expected level of Y-linked diversity that is consistent with what we observed. Our results are consistent with recent theory on interference selection, and suggest that interference is an important force in the evolution of  young Y chromosomes. In particular, rather than widespread silencing of Y-linked genes and their subsequent degeneration through increased neutral drift, our study suggests that when a large number of linked sites are subject to purifying selection, Y-chromosome degeneration in the early stages may be largely driven by the effects of selective interference.
\section*{Acknowledgments}

\bibliography{bibliography}
\end{document}