---
output: html_notebook
---
#### import polymorphurama data
```{r}
#workingdir
setwd("/Users/josh/Github/XYdiversity/temp/workspace")

#sex chromosomes
txx <- read.csv("txx.csv")
txy <- read.csv("txy.csv")

#autosomes
tm_all <- read.csv("tm_all_new.csv")
tf_all <- read.csv("tf_all_new.csv")

```

#### calculate the average weighted theta value for each (synsites*theta_syntot); calculate weighted average
```{r}
#new columns=SynSitestot*theta_syntot (for each cell)
#TX_X
txx$weighted_theta <- txx$SynSitestot * txx$theta_syntot            
txx_ave_weighted_theta=(sum(txx$weighted_theta))/(sum(txx$SynSitestot))
#TX_Y
txy$weighted_theta <- txy$SynSitestot * txy$theta_syntot            
txy_ave_weighted_theta=(sum(txy$weighted_theta))/(sum(txy$SynSitestot))
```


#### plot of X and Y weighted mean theta
```{r}
barplot(c(txx_ave_weighted_theta,
          txy_ave_weighted_theta),
          ylab=expression(paste("Nucleotide Diversity (average weighted ", theta, " )")),
          names.arg=c("TX_X","TX_Y"),
          ylim=c(0,0.005))
```

```{r}

```


#### Calculate X/A diversity ratios

```{r}
#import autosomal genes
#females
tf_all$weighted_theta <- tf_all$SynSitestot * tf_all$theta_syntot            
tf_all_ave_weighted_theta=(sum(tf_all$weighted_theta))/(sum(tf_all$SynSitestot))
#males
tm_all$weighted_theta <- tm_all$SynSitestot * tm_all$theta_syntot            
tm_all_ave_weighted_theta=(sum(tm_all$weighted_theta))/(sum(tm_all$SynSitestot))

#X/A
tx_x_auto=txx_ave_weighted_theta/tf_all_ave_weighted_theta
#Y/A 
tx_y_auto=txy_ave_weighted_theta/tf_all_ave_weighted_theta

```


#### Neutral expectations for an equal sex ratio, poisson offspring number (Kimura and Crow model)

```{r}
#set numbers of males and females
vm=2 #variance in female rep success
vf=2 #variance in female rep success
kf=2 #mean number of offspring of females
km=2 #mean number of offspring of males
Nf=100000 #numner of females
Nm=100000 #number of males
kbar=2 # total mean offspring number

# males and female Ne
Nem=(Nm*km-1)/(km-1+(vm/kbar)) #effect of male reproductive variance on female Ne
Nef=(Nf*kf-1)/(kf-1+(vf/kbar)) #effect of female reproductive variance on female Ne

# chromosome Ne's
NeA=(4*Nem*Nef)/(Nem+Nef)
NeX=(9*Nem*Nef)/((4*Nem)+(2*Nef))
NeY=Nem/2

YA_poisson=NeY/NeA
XA_poisson=NeX/NeA

```


#### Diversity predictions for a range of Nef/(Nem+Nef) ratios
```{r}
# expected X/A and Y/A curves as functions of the sex ratio:
curve((9*(-1 + x))/(8*(1 - x)*(-2 + x)), from=0, to=1, ylim=c(0.002,1.12),
      ylab="X/A", xlab="Nef/(Nem+Nef)")

curve(1/(8*x), from=0, to=1, add=T)


#X/A
XA_0.5=((9*Nef*0.5*(0.5 - 1))/(2*(0.5 - 2)))/(4*Nef*(1 - 0.5)*0.5); XA_0.5
XA_0.6=((9*Nef*0.6*(0.6 - 1))/(2*(0.6 - 2)))/(4*Nef*(1 - 0.6)*0.6); XA_0.6
XA_0.7=((9*Nef*0.7*(0.7 - 1))/(2*(0.7 - 2)))/(4*Nef*(1 - 0.7)*0.7); XA_0.7
XA_0.8=((9*Nef*0.8*(0.8 - 1))/(2*(0.8 - 2)))/(4*Nef*(1 - 0.8)*0.8); XA_0.8
XA_0.9=((9*Nef*0.9*(0.9 - 1))/(2*(0.9 - 2)))/(4*Nef*(1 - 0.9)*0.9); XA_0.9

#Y/A
YA_0.5=1/(8*0.5); YA_0.5
YA_0.6=1/(8*0.6); YA_0.6
YA_0.7=1/(8*0.7); YA_0.7
YA_0.8=1/(8*0.8); YA_0.8
YA_0.9=1/(8*0.9); YA_0.9

curve(((9*(-1 + x))/(8*(1 - x)*(-2 + x)))/0.75, from=0.5, to=1, ylim=c(0,1.5), lwd=2, 
      ylab="observed/expected", xlab="sex ratio")
curve((1/(8*x))/0.25, from=0.5, to=1, lwd=2, add=T)
abline(lty=2,h=1.13)
abline(lty=2,h=0.0004)

```

So, theoretically, the maximum Y/A reduction relative to neutrality is 0.5. We have a reduction of 0.00156 (+/- ).

```{r}
xa_given_y=0.0003 # y theta * 3
ya_given_x=0.00156 #x theta * 1/3
plot(c(tx_x_auto/0.75,XA_0.6/0.75, xa_given_y/0.75),
        ylim=c(0,1.5),
        ylab="X/A relative to neutral model", main="X/A for a sex ratio of 0.6", 
        xlab="expected                                                      observed")
curve((1/(8*x))/0.25, from=0.5, to=1, lwd=2, add=T)
abline(lty=2,h=1.13)
abline(lty=2,h=0.0004)


```


#### Plot empirical, simulated, and theoretically predicted diversity ratios for a range of Ne sex ratios, with CI's
```{r}

XA=c(tx_x_auto, XA_0.5, XA_0.6, XA_0.7, XA_0.8, XA_0.9)
YA=c(tx_y_auto, YA_0.5, YA_0.6, YA_0.7, YA_0.8, YA_0.9)
ChrRatios=cbind(XA,YA)

barplot(as.matrix(ChrRatios), beside=TRUE, col=c("darkgreen", "black", "gray22","gray73","gray87","gray93","darkgreen","black", "gray22","gray73","gray87", "gray93"),
        ylab = expression(paste("Nucleotide Diversity ( ", theta, " ) ratio")),
        ylim=c(0,1.14))
legend("topright", c("observed", "Nf/(Nf+Nm) = 0.5", "Nf/(Nf+Nm) = 0.6", "Nf/(Nf+Nm) = 0.7", "Nf/(Nf+Nm) = 0.8", "Nf/(Nf+Nm) = 0.9"), fill=c("darkgreen", "black", "gray22","gray73","gray87", "gray93"), box.lty=0, cex=0.7)

# empirical XA confidence intervals
segments(x0=1.5,y0=0.7,x1=1.5,y1=0.9,lwd=0.75)
arrows(x0=1.5,y0=0.7,x1=1.5,y1=0.9,lwd=0.75,angle = 90, code = 3, length = 0.04)

# simulated XA confidence intervals
#segments(x0=2.5,y0=0.7,x1=2.5,y1=0.8,lwd=0.75)
#arrows(x0=2.5,y0=0.7,x1=2.5,y1=0.8,lwd=0.75,angle = 90, code = 3, length = 0.04)

# empirical YA confidence intervals
segments(x0=8.5,y0=0.01,x1=8.5,y1=0.04,lwd=0.75)
arrows(x0=8.5,y0=0.01,x1=8.5,y1=0.04,lwd=0.75,angle = 90, code = 3, length = 0.04)

# simulated YA confidence intervals
#segments(x0=10.5,y0=0.01,x1=10.5,y1=0.04,lwd=0.75)
#arrows(x0=10.5,y0=0.02,x1=10.5,y1=0.07,lwd=0.75,angle = 90, code = 3, length = 0.04)

```


```{r}
#0.25*(exp(-(0.02/((2*0.002)+0)))
#bgs_exp=0.00168449

BGS_YA=0.0054
SEXRATIO_YA=YA_0.6
OBS_YA=0.018
  
#barplot(c(OBS_YA/YA_0.6,BGS_YA/YA_0.6), ylim=c(0,0.2), col="gray52", ylab="Y/A ratio relative to the neutral prediction",
#        names.arg=c("observed", "BGS theory prediction"), cex.names=0.75)
#segments(x0=0.7,y0=0.009/YA_0.6,x1=0.7,y1=0.03/YA_0.6,lwd=0.75)
#arrows(x0=0.7,y0=0.009/YA_0.6,x1=0.7,y1=0.035/YA_0.6,lwd=0.75,angle = 90, code = 3, length = 0.04)


par(las=1)
barplot(c(OBS_YA/YA_0.6,BGS_YA/YA_0.6), ylim=c(0,0.2), col="gray52",ylab=expression(paste(pi/pi[0])),
        names.arg=c("observed", "BGS theory"), cex.names=0.75)
segments(x0=0.7,y0=0.009/YA_0.6,x1=0.7,y1=0.035/YA_0.6,lwd=0.75)
arrows(x0=0.7,y0=0.009/YA_0.6,x1=0.7,y1=0.035/YA_0.6,lwd=0.75,angle = 90, code = 3, length = 0.04)


```

